setup.py

```diff
@@ -16,6 +16,7 @@
         'chardet ; python_version > "3.0"',
         'cryptography',
         'sortedcontainers',
+        'python-bidi',
```

Comment:
Is python-bidi still maintained?  Most development work was done 2014 and 2015 and only one release was made since then in 2019.

Reply:
It is licensed with GNU Lesser General Public License v3.0, so we can also copy paste the library into pdfminer.six. If we are expecting extensive modifications we can also create a new pdfminer group project and release the package ourself. 

I would suggest starting with copy the 2 source files.